crates/re_types/tests/points2d.rs

```diff
@@ -0,0 +1,49 @@
+use re_types::{archetypes::Points2D, components};
+
+#[test]
+fn roundtrip() {
+    // TODO(cmc): (de)serialization roundtrips
+
+    let arch = Points2D::new([(1.0, 2.0), (3.0, 4.0)])
+        .with_radii([42.0, 43.0])
+        .with_colors([0xAA0000CC, 0x00BB00DD])
+        .with_labels(["hello", "friend"])
+        .with_draw_order(300.0)
+        .with_class_ids([126, 127])
+        .with_keypoint_ids([2, 3])
+        .with_instance_keys([u64::MAX - 1, u64::MAX]);
+
+    let expected = Points2D {
+        points: vec![
+            components::Point2D::new(1.0, 2.0), //
```

Comment:
mysterious comments you have there all over the place

Reply:
the usual trick to prevent rustfmt from collapsing lists on one line